feature_selection_utils.py

```diff
@@ -1,33 +1,61 @@
 import matplotlib.pyplot as plt
 import numpy as np
 from scipy.stats import pearsonr
+from sklearn.decomposition import PCA
 
-def calculate_features_target_correlation(data,features,target,method):
+
+def calculate_features_target_correlation(data, features, target, method):
     """
-    This function receives the pandas dataframe, the list of features, and a string that representing the name of the target column in data frame.
+    This function receives the pandas dataframe, the list of features, and a string that representing the name of the
+    target column in data frame.
     The output is the correlation of each numeric feature vector with the target values and their bar plot.
     method specifies the type of correlation method.
+    :type data: pandas dataframe
+    :param data: data set to be analysed
+    :type features: list of strings
+    :param features: the list of features that are being analysed
+    :type target: string
+    :param target: the column of data set that is treated as the output for prediction
+    :type method: string
+    :param method: specisifes the method for finding the correlation coefficients and can be "Pearson" or "Cross-cor"
```

Comment:
same here. Maybe you could run it on PyCharm to help you out with the count. It gets tricky, I understand